types/kakaomaps/kakaomaps-tests.ts

```diff
@@ -4,3 +4,28 @@ const options = {
     level: 3
 };
 const map = new kakao.maps.Map(container, options);
+
+// example: https://apis.map.kakao.com/web/sample/donut/
+const polygon = new kakao.maps.Polygon({ map });
+
+const path = [
+	new kakao.maps.LatLng(33.45086654081833, 126.56906858718982),
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/kakaomaps/kakaomaps-tests.ts;
  Line: 12;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.